assets/css/js_interop.css

```diff
@@ -319,12 +319,82 @@ solely client-side operations.
   @apply text-green-bright-400;
 }
 
-[data-el-session]:is([data-js-view="code-zen"], [data-js-view="presentation"])
+[data-el-session]:is(
+    [data-js-view="code-zen"],
+    [data-js-view="presentation"],
+    [data-js-view="custom"]
+  )
   [data-el-views-disabled] {
   @apply hidden;
 }
 
-[data-el-session]:not([data-js-view="code-zen"], [data-js-view="presentation"])
+[data-el-session]:not(
+    [data-js-view="code-zen"],
+    [data-js-view="presentation"],
+    [data-js-view="custom"]
+  )
   [data-el-views-enabled] {
   @apply hidden;
 }
+
+/* Custom view */
+
+/* outputs */
+[data-js-view="custom"]:not([data-js-output="true"]) [data-el-output] {
+  @apply hidden;
+}
+
+/* section */
+[data-el-session][data-js-view="custom"]:not([data-js-section="true"])
+  [data-el-section-headline],
+[data-el-session][data-js-view="custom"]:not([data-js-section="true"])
+  [data-el-section-subheadline],
+[data-el-session][data-js-view="custom"]:not([data-js-section="true"])
+  [data-el-section-subheadline-collapsed] {
+  @apply hidden;
+}
+
+/* markdown */
+[data-el-session][data-js-view="custom"]:not([data-js-markdown="true"])
+  [data-el-cell][data-type="markdown"],
+[data-el-session][data-js-view="custom"]:not([data-js-markdown="true"])
+  [data-el-actions],
+[data-el-session][data-js-view="custom"]:not([data-js-markdown="true"])
+  [data-el-insert-buttons] {
+  @apply hidden;
+}
+
+/* results */
+[data-el-session][data-js-view="custom"]:not([data-js-results="true"])
+  [data-el-actions],
```

Comment:
What are the results in this case? What is the difference between results and output?

Reply:
The output is the result of the evaluation while the **results** enables the action buttons above the cell, i think they're both the same @josevalim

Reply:
We don’t need to disable the action buttons above the cell, so we can remove the results bit. :) we can disable the whole code cell later, but not a concern for now. :)